code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1874,12 +1900,6 @@
 	taste_description = "a wall of bricks"
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 
-/datum/glass_style/drinking_glass/bacchus_blessing
-	required_drink_type = /datum/reagent/consumable/ethanol/bacchus_blessing
-	name = "Bacchus' Blessing"
-	desc = "You didn't think it was possible for a liquid to be so utterly revolting. Are you sure about this...?"
-	icon_state = "glass_brown2"
-
```

Comment:
Why is this being renoved?

Reply:
Fixed with 51b688baf4ab006ba56aa3c25d003ffcad88542a thanks for pointing it out!